File: Java/Factorial.java
import java.util.Scanner;
class Factorial
{  
 public static void main(String args[])
 {  
   Scanner sc = new Scanner(Syatem.in);
   System.out.println("Enter the Number ");
   int number =sc.nextInt();
  int i, fact =1;  
  for(i=1;i<=number;i++)
  {    
      fact=fact*i;    
  }    
  System.out.println("Factorial of "+number+" is: "+fact);    
 }  
}  

File: CPP/Data Structures/math/factorial.cpp
#include<bits/stdc++.h>
using namespace std;

int factorial(int n)
{
    int fac=1;
    for(int i=1;i<=n;i++)
    {
        fac=fac*i;
    }
    return fac;
}
int main()
{
    int x;
    cout<<"enter the number";
    cin>>x;
    cout<<factorial(x);
}